symfony/usr/local/share/symfony/symfony_functions.sh

```diff
@@ -58,12 +98,18 @@ do_database_build() {
 
 do_database_install() {
   do_database_schema_create
-  do_database_migrations_mark_done
+  if uses_symfony_doctrine_mode_migrations; then
+    do_database_migrations_mark_done
```

Comment:
I think we should run the migration script instead of running the database schema create. The reason for this is that some application will have migrations that are not 100% in sync with the doctrine generated schema, and that's exactly why they use the migrations instead of schema:update.